GOVERNANCE.md

```diff
@@ -129,7 +129,16 @@ When an agenda item has appeared to reach a consensus the moderator
 will ask "Does anyone object?" as a final call for dissent from the
 consensus.
 
+Agenda items should be tagged at least 24 hours in advance. If an item
+is added less than 24 hours before the meeting it cannot be brought to
+a vote.
+
 If an agenda item cannot reach a consensus a TC member can call for
 either a closing vote or a vote to table the issue to the next
 meeting. The call for a vote must be seconded by a majority of the TC
-or else the discussion will continue. Simple majority wins.
+or else the discussion will continue. Simple majority wins, those not
+in attendance abstain.
```

Comment:
Disagree. We should be able to email in our vote if circumstances don't allow us to attend.